packages/component/src/Activity/private/hasFeedbackLoop.ts

```diff
@@ -1,17 +1,35 @@
 import { type WebChatActivity } from 'botframework-webchat-core';
-import { literal, object, safeParse, string, type InferOutput } from 'valibot';
+import { literal, string, object, safeParse, type InferOutput, nullish } from 'valibot';
 
-const activityWithFeedbackLoopSchema = object({
+const activityWithFeedbackLoopSchemaWithDisclaimer = object({
   channelData: object({
     feedbackLoop: object({
-      disclaimer: string(),
+      disclaimer: nullish(string()),
```

Comment:
```suggestion
      disclaimer: optional(string()),
```

We hardly use/allow `null` in Web Chat.